src/main/webapp/authentication-app/package-lock.json

```diff
@@ -0,0 +1,17284 @@
+{
```

Comment:
Is the package-lock.json file necessary? I think it might be something we put in the .gitignore file unless it's necessary

Reply:
Im not sure to be honest. It was added to my commit because I ran into an event error and when looking around online I found that many people were saying it was a bug with the react script and showed some commands for a fix. I did those commands, not sure what they did, but it fixed the error and modified those files.